app/models/user.rb

```diff
@@ -76,6 +76,12 @@ def full_name
     "#{first_name} #{last_name}"
   end
 
+  def enabled_preferences_list
+    preferences.where(restriction: true).pluck(:name, :description).map { |name, description|
+      "- #{name}: #{description}"
+    }.join("\n")
+  end
```

Comment:
As we discussed, Rails best practices suggest keeping models focused on data, while formatting or presentation logic is better suited for a service. So it would be better to try to move it to the RecipeGeneratorService.

Reply:
I would prefer to have the logic on the service and exclude it on the reek.yml (so that the linters does not fail with FeatureEnvy: RecipeGeneratorService#enabled_preferences) rather have it on user model.